### PR TITLE
Removed GTK2 and Cairo support

### PR DESCRIPTION
**Pull request by kekekeks**
:fire: :fire: :fire:

### CODE REVIEW

**Comment by JimSEOW** [comment]
@kekekeks I would like your feedback on providing[ XAML's 3D DisplayPanel ](https://github.com/Microsoft/xaml-standard/issues/183#issuecomment-332392234)using SkiaSharp's OpenGLES

**Comment by kekekeks** [comment]
@JimSEOW I'm not sure what do you mean by using SkiaSharp for 3D. It's a 2D graphics library that can use OpenGL (ES) for hardware-acceleration and that's it. 

If you want to know our plans on supporting 3D widgets, then I can tell you that it will almost surely be implemented using OpenGL with ANGLE as translation layer for UWP/Win32.

In fact, one can start implementing it right now using OpenTK and convert hw-texture to a regular bitmap, so it could be used with any platform. I also think, that it could as well be a netstandard library with views for different UI frameworks, OxyPlot guys are doing that and it works out just fine.

File: samples/ControlCatalog/MainWindow.xaml.cs [hunk 1]
```diff
@@ -2,18 +2,43 @@
...
+        private int _wtf;
         public MainWindow()
         {
-            this.InitializeComponent();
+            //this.InitializeComponent();
             this.AttachDevTools();
             //Renderer.DrawDirtyRects = Renderer.DrawFps = true;
         }
 
+        async Task Wtf()
```
Comment (grokys) [inline samples/ControlCatalog/MainWindow.xaml.cs]: Did you mean to commit these changes?

**Review by grokys — COMMENTED** [review]
LGTM except what are the changes in ControlCatalog for?

File: samples/ControlCatalog/MainWindow.xaml.cs (hunk 1 above)
Comment (kekekeks) [reply samples/ControlCatalog/MainWindow.xaml.cs]: Nope, a leftover from trying to reproduce issues with OxyPlot in controlled environment

**Comment by JimSEOW** [comment]
@kekekeks  Thanks for your feedback.

It is extremely exciting that the PR Remove-GTKebab is making Avalonia lean.
I am new to Avalonia for UWP. 

Could you please provide a Wiki Page **(Eventually) the milestone towards UWP**. Once that is reached, a sample will be great to inspire the user.

I have tried OpenTKCore and ANGLE.WindowsStore from Nuget

There are 3 options to bring DirectX3D to UWP's XAML.
Using Image as you have suggested is one of the recommended ways.
The alternative is UWP XAML Interop using SwapChainPanel.

With SkiaSharp, (for UWP), not only one gets 2D, one could also access to the SkiaSharp.View's [SwapChainPanel ](https://github.com/mono/SkiaSharp/blob/master/source/SkiaSharp.Views/SkiaSharp.Views.UWP/AngleSwapChainPanel.cs)for XAML-InterOp for 3D

This is what I meant by **in UWP, SkiaSharp provides 2 in 1 (2D and 3D displays in one Nuget).**

**Review by grokys — APPROVED** [review]

**Pull request merged**